store/reducers/steps/index.tsx

```diff
@@ -12,7 +12,7 @@ export const defaultStepsState: StepsStore = {
   steps: [
     {
       label: 'steps.userType',
-      component: 'TipUtilizator',
+      component: <UserTypeForm />
```

Comment:
we should not keep the component in the redux store as we need to be keep the store serializable.
i recommend keeping a string key here, and in the component where the `steps` value is used to [dynamic import](https://nextjs.org/docs/advanced-features/dynamic-import) the `UserTypeForm` if `step.component === "TipUtilizator"`

Reply:
![Screenshot 2022-03-01 at 16 55 04](https://user-images.githubusercontent.com/6429818/156191938-11c5a7a4-3fff-44f8-8c8a-5008ed442d33.png)
this should help with performance too

Reply:
Done :heavy_check_mark: